### DISCUSSION TITLE
How can I delete my account?

### DISCUSSION THREAD

**Discussion by mr-cheffy** [supabase/supabase#18763] [Questions] [answered]
The title says it all.

**Comment by GaryAustin1** [comment]
You need to contact support to delete your Supabase account.  The dashboard is best or support@supabase.io .

**Reply by zoeleu** [reply]
Come on... This is ridiculous...

**Reply by GaryAustin1** [reply, marked as answer]
This changed in the last couple of months....
![image](https://github.com/supabase/supabase/assets/54564956/2050c39d-3112-4e22-bc12-2a8fabb6d58f)
